packages/share_plus_web/test/test_driver/share_plus_web_integration.dart

```diff
@@ -0,0 +1,36 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:mockito/mockito.dart';
+import 'dart:html' as html;
+
+import 'package:share_plus_web/share_plus_web.dart';
+
+class _MockWindow extends Mock implements html.Window {}
+
+class _MockNavigator extends Mock implements html.Navigator {}
+
+void main() {
```

Comment:
I think this should be in `/test/lib/main.dart` but not 100% sure. What do you think @jpnurmi ? as you did the integration tests for connectivity_plus_web you may know better than I do.